src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Configurations/BeanstalkApplicationConfiguration.cs

```diff
@@ -7,5 +7,20 @@ public class BeanstalkApplicationConfiguration
     {
         public bool CreateNew { get; set; }
         public string ApplicationName { get; set; }
+
+#nullable disable warnings
```

Comment:
Can you add a comment why we disable the warning?

Reply:
I added a comment for all classes that are doing this.